string name
float64 axisX
float64 axisY
float64 axisZ
float64 angle
